### DISCUSSION TITLE
Logging limited to 64K

### DISCUSSION THREAD

**Discussion by swhiteman** [dannagle/PacketSender#364] [Q&A] [answered]
It seems like only the first 64K of data is logged for inbound requests. (I can see more data using Wireshark.) Is there a way to up this limit?

**Comment by dannagle** [comment, marked as answer]
Packet Sender will attempt to concatenate all the data until there is a pause in the stream. Perhaps you are stretching a limitation in the code. 

Do you see all the data in the viewer if you enable persistent TCP?

**Comment by swhiteman** [comment]
Aha, with persistent TCP I can see it all. Thanks.
